gtfs/spec/en/reference.md

```diff
@@ -64,7 +64,7 @@ Presence conditions applicable to fields and files:
 - **Float** - A floating point number.
 - **Integer** - An integer.
 - **Phone number** - A phone number.
-- **Time** - Time in the HH:MM:SS format (H:MM:SS is also accepted). The time is measured from "noon minus 12h" of the service day (effectively midnight except for days on which daylight savings time changes occur). For times occurring after midnight, enter the time as a value greater than 24:00:00 in HH:MM:SS local time for the day on which the trip schedule begins. <br> *Example: `14:30:00` for 2:30PM or `25:35:00` for 1:35AM on the next day.*
+- **Time** - Time must be either in the HH:MM:SS or H:MM:SS format. The time must be measured in the hours from midnight on the day on which the trip began using the time zone specified by `agency.agency_timezone`. For trips which span midnight in the timezone specified by `agency.agency_timezone`, times after midnight should be greater than 24:00:00. <br> *Example: `14:30:00` for 2:30PM or `25:35:00` for 1:35AM on the next day.*
```

Comment:
> on the day on which the trip began

With a trip having 3 `stop_times` at `25:00:00`, `25:30:00` & `26:00:00`, running on service day `2022-06-06`, would the time be measured from `2022-06-06T00:00:00+$offset` or from `2022-06-07T00:00:00+$offset`? I think the previous phrasing "service day" is more precise.

Reply:
I also think removing the "noon minus 12h" definition a) breaks existing datasets, and b) doesn't make processing GTFS datasets easier and less error-prone. I proposed alternative changes here: https://github.com/google/transit/issues/325#issuecomment-1126346838